README.md

```diff
@@ -1 +1,81 @@
-# Connect-landing-page
\ No newline at end of file
+
+# Project Name: Smart Online Coding School
+
+>**Smart is a platform that provides student from all around the world an opportunity to learn coding without having to worry distance, Fees and other hinders that can easily be found in tradition school systems.(This is A capstone project done at Microverse).**
+
+## Built With
+
+- HTML and CSS
+- Javascript
+- npm(Node.js)
+- Font-awesome
+- SASS Preprocessor( SCSS )
+
+## Live Demo
+
+[**See Project 🚀**](https://davezag.github.io/Smart-Online-school-capstone-project/)
+
+<!-- 
+## Video Demonstration
+
+[**Watch**](https://) -->
```

Comment:
- [ ] Please record a video presenting the features of the project and add the link to the video in the pull request summary.

- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
- The video contains a demo of the project, showing the pages in different screen sizes.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.